app/src/main/java/com/bogdan/codeforceswatcher/feature/users/redux/Sorting.kt

```diff
@@ -0,0 +1,21 @@
+package com.bogdan.codeforceswatcher.feature.users.redux
+
+import com.bogdan.codeforceswatcher.model.User
+import com.bogdan.codeforceswatcher.room.DatabaseClient
+
+object Sorting {
```

Comment:
Good idea 👍You can improve it further by making it an extension of `fun List<User>.sort(sortType: UsersState.SortType)`.